packages/service-library/src/servicelib/logging_utils.py

```diff
@@ -141,7 +141,44 @@ def test_logger_propagation(logger: logging.Logger) -> None:
     logger.debug(msg, "debug")
 
 
-def _log_arguments(
+class LogExceptionsKwargsDict(TypedDict, total=True):
```

Comment:
as usual, I prefer dataclass. also I'm reading more and more that typedict is slowly replaced by it.
and BTW: total=True is the default

Reply:
@sanderegg this is just for mypy to make sure signature errors are caught. Note how is it used. I do not see advantage here in creating a dataclass and then converting it again to a dict. Or perhaps I am missing something?